### ISSUE TITLE
Month tag from acm digital library crash the parser

### ISSUE THREAD

**Issue by SebaRF** [MaikelH/BibtexLibrary#7]
The bib file:

@article{Miller:2015:ISI:2742853.2724759,
 author = {Miller, Craig S. and Connolly, Randy},
 title = {Introduction to the Special Issue on Web Development},
 journal = {Trans. Comput. Educ.},
 issue_date = {March 2015},
 volume = {15},
 number = {1},
 month = mar,
 year = {2015},
 issn = {1946-6226},
 pages = {1:1--1:5},
 articleno = {1},
 numpages = {5},
 url = {http://doi.acm.org/10.1145/2724759},
 doi = {10.1145/2724759},
 acmid = {2724759},
 publisher = {ACM},
 address = {New York, NY, USA},
 keywords = {Web development},
}

As you can see the "month" tag comes without "{}" wich crash the parser, i'd love to bring a solution if i knew how.

**Comment by MaikelH** [comment]
Yes, this is a known problem in the parser currently. The grammar needs to be updated to reflect this, but the problem is that it is pretty difficult to do this of the ambiguity in bibtex itself.

I am currently pretty busy, but I will see if I can get some work done the coming week.
